frontend/scripts/serve.js

```diff
@@ -1,17 +1,26 @@
 // Serve app locally without building with webpack, e.g. for e2e
+const path = require("path");
+
 const connectHistoryApiFallback = require("connect-history-api-fallback");
 const express = require("express");
 const { createProxyMiddleware } = require("http-proxy-middleware");
 
-const devServerConfig = require("../config/dev-server.js");
+const dotEnvPath = path.resolve(process.cwd(), ".env.local");
+require("dotenv").config({
```

Comment:
how was this imported before, out of curiosity, if it wasn't already here?

Reply:
Not quite sure why we did this, but `dev-server.js` previously imported `webpack.config.js`, which imports `dotenv`.